src/plugins/loan-channel-query-map.js

```diff
@@ -238,7 +238,7 @@ export default {
 				{
 					id: 60,
 					url: 'loans-that-are-almost-funded',
-					queryParams: 'status=fundRaising&riskRating=0,5',
+					queryParams: 'status=fundRaising&riskRating=0,5&sortBy=amountLeft',
```

Comment:
amountLeft is now enabled on lend/filter flss so this makes it compatible.

Reply:
This channel will need a `flssLoanSearch` if want it to be served by FLSS.

Reply:
Good call. We'll do another pass soon on this file soon to solidify the FLSS queries on our category page and we can include this category then.